base/install_motor.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -ex
+
+git clone --depth 1 --branch ${MOTOR_VERSION} \
+    https://github.com/epics-modules/motor
+
+cd motor/modules
+
+git submodule update --init --depth 1 -j ${JOBS} \
+    motorMotorSim
+
+module_releases="
+MOTOR=${EPICS_MODULES_PATH}/motor
+MOTOR_MOTORSIM=${EPICS_MODULES_PATH}/motor/modules/motorMotorSim
+"
+echo "$module_releases" >> ${EPICS_MODULES_PATH}/../RELEASE
+
+cd ../configure
+
+echo "
+EPICS_BASE=${EPICS_BASE_PATH}
+MOTOR=${EPICS_MODULES_PATH}/motor
+ASYN=${EPICS_MODULES_PATH}/asyn
+SNCSEQ=${EPICS_MODULES_PATH}/seq
+AUTOSAVE=${EPICS_MODULES_PATH}/autosave
+BUSY=${EPICS_MODULES_PATH}/busy
+IPAC=${EPICS_MODULES_PATH}/ipac
+" > RELEASE.local
+
+ln -s RELEASE.local RELEASE_PRODS.local
+ln -s RELEASE.local RELEASE_LIBS.local
+
+echo "
+BUILD_IOCS=YES
```

Comment:
I recall you telling me we don't use IOCs directly from the support motor modules. Is that right? If so, I'd rather have this set to `NO` instead. We don't need to build unused software.